base/db/R/get.trait.data.R

```diff
@@ -27,11 +27,12 @@
 ##' @param trait.names Character vector of trait names to search. If
 ##'   `NULL` (default), use all traits that have a prior for at least
 ##'   one of the `pfts`.
+##' @param update logical: Rerun the db if result files already exist?
 ##' @return list of PFTs with update posteriorids
 ##' @author David LeBauer, Shawn Serbin, Alexey Shiklomanov
 ##' @export
 get.trait.data <- function(pfts, modeltype, dbfiles, database, forceupdate,
-                           trait.names = NULL) {
+                           trait.names = NULL, update = FALSE) {
```

Comment:
Could you remove the addition of `update` as an argument to this function? As far as I can tell the only place it is used is where you added it to `query.priors` and `get.trait.data.pft`, and in both cases this appears to be a mistake.